pycloudlib/oci/cloud.py

```diff
@@ -135,13 +135,7 @@ def get_instance(self, instance_id):
             raise Exception(
                 "Unable to retrieve instance with id: {} . "
                 "Is it a valid instance id?".format(instance_id)) from e
-        state = instance_data.lifecycle_state
```

Comment:
CI is failing because `instance_data` is now unused; this LGTM otherwise.

Reply:
I removed the variable but want to keep the call to make sure if we're calling `get_instance` manually that we're passing in a valid `instance_id`